### DISCUSSION TITLE
Update-2023-07-24 Double Spirograph

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#231] [Announcements]
A "double" spirograph setup contains two gears rolling against each other. This setup is not physically possible. The white designs below are a single curve.

![Screen Shot 2023-07-24 at 12 56 55 PM](https://github.com/kaveh808/kons-9/assets/103860793/b18621a5-979c-49d4-8b6b-194f1a51ba23)
![Screen Shot 2023-07-24 at 12 58 45 PM](https://github.com/kaveh808/kons-9/assets/103860793/aafb4b58-4e57-4f73-8cd3-1adda8b409d6)
![Screen Shot 2023-07-24 at 12 57 33 PM](https://github.com/kaveh808/kons-9/assets/103860793/119be23e-2191-4228-9759-a741239120a1)

**Comment by kayomarz** [comment]
This is great! As a kid I used to spend hours drawing designs with a Spirograph.

**Reply by kaveh808** [reply]
Have fun with the demos and see what you can generate.
